delfin/tests/e2e/README.md

```diff
@@ -0,0 +1,26 @@
+# Introduction
+This folder contains end to end, automated, testing scripts for Delfin.
+
+These tests are using [Robot Framework](https://robotframework.org/) for automation and report generation.
+
+The end-to-end tests are run against a test driver provided in the path `delfin/tests/e2e/testdriver`.
+This test driver uses, included storage details in file `delfin/tests/e2e/testdriver/storage.json` for storage simulation when testing.
+
+# Supported OS
+Ubuntu 18.04
+
+# Prerequisite
```

Comment:
prerequisite for standalone installer applicable here. either copy those steps here, or we link to that document

Reply:
Done